examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Examples
+
+In order to run these examples, you will need a running Apache-Pulsar backend. You can start one with:
+
+```bash
+$ sudo docker run -it \
+  -p 6650:6650 \
+  -p 8080:8080 \
+  --mount source=pulsardata,target=/pulsar/data \
+  --mount source=pulsarconf,target=/pulsar/conf \
+  apachepulsar/pulsar:2.7.0 \
+  bin/pulsar standalone
```

Comment:
```suggestion
sudo docker run -it \
  -p 6650:6650 \
  -p 8080:8080 \
  --mount source=pulsardata,target=/pulsar/data \
  --mount source=pulsarconf,target=/pulsar/conf \
  apachepulsar/pulsar:2.10.0 \
  bin/pulsar standalone
```

Use a mainatined version and IIRC we are targeting to 2.10.0 compatible now.

Also, removing `$` so that it's easy to copy, paste and run.

Reply:
Remove `sudo`, you may want to use `-u $(id -u):(id -g)`?

Reply:
It's an antipattern to not use sudo with Docker and a sever security vulnerability leading to potential privilege escalation from inside the container. To bot mention potential PAM MFA authentication when calling sudo.

Having said that, each user is responsible of how they call Docker, so we can remove the sudo call.